pennylane/ops/qubit/state_preparation.py

```diff
@@ -19,7 +19,10 @@
 
 import pennylane as qml
 from pennylane.operation import AnyWires, Operation
-from pennylane.templates.state_preparations import BasisStatePreparation, MottonenStatePreparation
+from pennylane.templates.state_preparations import (
+    BasisStatePreparation,
+    MottonenStatePreparation,
+)
```

Comment:
Out of curiosity was this a recommended change by black while you were linting ?

Reply:
Sorry, this has been caused by not specifying `-l 100` when executing black. I have reverted the change.